Makefile

```diff
@@ -89,7 +89,7 @@ prepare-e2e-tests: build docker push
 
 	@cat test/operator.yaml | sed "s~image: jaegertracing\/jaeger-operator\:.*~image: $(BUILD_IMAGE)~gi" >> deploy/test/namespace-manifests.yaml
 
-	# ClusterRoleBinding is created in test codebase because we don't know service account namespace
+	@# ClusterRoleBinding is created in test codebase because we don't know service account namespace
```

Comment:
Is `@` needed? It's just a comment

Reply:
Otherwise it is seen in the console logs:

```
Writing manifest to image destination
Storing signatures
# ClusterRoleBinding is created in test codebase because we don't know service account namespace
Creating namespace default
Running Cassandra end-to-end tests...
```